Adapter/PhpcrOdmAdapter.php

```diff
@@ -154,7 +169,21 @@ public function createAutoRoute(UriContext $uriContext, $contentDocument, $autoR
      */
     public function createRedirectRoute(AutoRouteInterface $referringAutoRoute, AutoRouteInterface $newRoute)
     {
-        $referringAutoRoute->setRedirectTarget($newRoute);
+        switch ($this->redirectTarget) {
+            case self::REDIRECT_CONTENT:
+                $target = $newRoute->getContent();
```

Comment:
The main thing that concerns me is the change of behavior - the option is there to preserve the existing behavior, if we now set the redirect target as the content, and the user is handling the redirect as if it were always a Route object, then it's a BC break ?